discovery/v1.js

```diff
@@ -0,0 +1,331 @@
+/**
+ * Copyright 2015 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var util = require('util');
+var requestFactory = require('../lib/requestwrapper');
+var BaseService = require('../lib/base_service');
+var pick = require('object.pick');
+
+/**
+ *
+ * @param options
+ * @constructor
+ */
+function DiscoveryV1(options) {
+  BaseService.call(this, options);
+
+  // Check if 'version_date' was provided
+  if (typeof this._options.version_date === 'undefined') {
+    throw new Error('Argument error: version_date was not specified, use DiscoveryV1.VERSION_DATE_2016_07_11');
+  }
+  this._options.qs.version = options.version_date;
+}
+
+util.inherits(DiscoveryV1, BaseService);
+DiscoveryV1.prototype.name = 'discovery';
+DiscoveryV1.prototype.version = 'v1-experimental';
```

Comment:
BTW, I'm not seeing the change in the PR - perhaps you put it in a different branch or forgot to push it to github?